pending/netbeans_ide.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2015 Andrew Ziem
+    http://bleachbit.sourceforge.net
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	@app     Netbeans 8.x
+	@os      Windows
+	@version 0.1.0
+-->
+<cleaner id="netbeans_ide" os="windows">
+  <label>NetBeans IDE</label>
+  <description>Integrated development environment (IDE)</description>
+  <running type="exe">netbeans.exe</running>
+  <running type="exe">netbeans64.exe</running>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs (keep latest log)</description>
+    <!-- This application used in software development so deleting all logs is a bad idea -->
+    <action command="delete" search="walk.all" path="$APPDATA\NetBeans\8.*\var\log\" regex="\w+(\.\d+)$"/>
```

Comment:
I will change this to all logs because when I think about it I can just NOT clear the cache after a problem occurs :bulb: 
Otherwise other users might complain about the fact that trash remains